utreexo/Cargo.lock

```diff
@@ -0,0 +1,6 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Libraries generally do not put Cargo.lock under version control. This is only useful for applications to lock down all dependencies. Libraries can be used with differently locked down applications, so they only specify versions per SemVer rules to allow upgrades.